electron-builder.cjs

```diff
@@ -84,6 +129,36 @@ const config = {
       schemes: ['decentraland'],
     },
   ],
+  afterPack: async context => {
+    if (process.platform === 'darwin' && process.env.MODE === 'production') {
```

Comment:
for MacOS, we could remove remaining development files with this `strip` command